openslides_backend/permissions/permissions.py

```diff
@@ -24,11 +24,19 @@ def is_ok(self, user_oml: str) -> bool:
         return self.numbers.get(user_oml, 0) >= self.number
 
 
-class CommitteeManager:
+class CommitteeManagementLevel(str, Enum):
     """ 2nd Permission Type, implemented as User.committee_as_manager_ids """
+    MANAGER = "can_manage_committees"
 
-    def __str__(self) -> str:
-        return "Committee Manager Right"
+    def __init__(self, cml: str):
+        super().__init__()
+        self.numbers = {
+            "can_manage_committees": 1,
+        }
+        self.number: int = self.numbers.get(cml, 0)
+
+    def is_ok(self, user_cml: str) -> bool:
```

Comment:
I don't like the name. I would suggest another approach: overriding the comparison operators. You could do that in a base class for committee and organisation management level and redefine `<`, `<=`, `>`, `>=` based on the numbers defined in the subclasses. That would be a very clean solution and would make handling these permissions easy. Of course this requires a simple order of the management levels, but that's currently the case.

Reply:
Talked about it and not needed for now. New name suggestion: `is_at_least`

Reply:
After all I realized the version with operators, because the name-version seems always to be misleading.